packages/edit-site/src/components/global-styles/font-library-modal/fonts-grid.js

```diff
@@ -29,30 +25,26 @@ function FontsGrid( { title, children, pageSize = 32 } ) {
 	}, [ lastItem ] );
 
 	return (
-		<div className="font-library-modal__fonts-grid">
-			<VStack spacing={ 0 }>
-				{ title && (
-					<>
-						<Text className="font-library-modal__subtitle">
-							{ title }
-						</Text>
-						<Spacer margin={ 2 } />
-					</>
-				) }
-				<div className="font-library-modal__fonts-grid__main">
-					{ items.map( ( child, i ) => {
-						if ( i === itemsLimit - 1 ) {
-							return (
-								<div key={ child.key } ref={ setLastItem }>
-									{ child }
-								</div>
-							);
-						}
-						return <div key={ child.key }>{ child }</div>;
-					} ) }
-				</div>
-			</VStack>
-		</div>
+		<VStack className="font-library-modal__fonts-grid" as="fieldset">
+			{ title && (
+				<legend className="font-library-modal__fonts-grid-legend">
```

Comment:
I'm not sure using a H2 heading _within a fieldset legend_ makes much sense. It is true that semantically a fieldset elemetn has a native role of `group` but it's a bit overkill. I think headeings and lists already provide the necessary semantics and information to users. I'd ratehr remove the fieldset and legend entirely.

Reply:
Fixed with [bdbe0bc](https://github.com/WordPress/gutenberg/pull/58834/commits/bdbe0bcd2704f3a69e24d054d18fe1f3eb968f2c)